public/app/features/dashboard/components/GenAI/utils.ts

```diff
@@ -123,3 +41,13 @@ export function getDashboardChanges(dashboard: DashboardModel): {
     migrationChanges: jsonDiff(originalDashboard, dashboardAfterMigration),
   };
 }
+
+/**
+ * Check if the LLM plugin is enabled and configured.
+ * @returns true if the LLM plugin is enabled and configured.
+ */
+export async function isLLMPluginEnabled() {
```

Comment:
re-added this to make our feature easier to test 👍 alternative here is to default to mocking entire openai llm api which also would work but may be more fragile / dependent on increased maintenance to keep up to date with llm api